code/datalab-app/src/app/components/welcome/HeroBar.js

```diff
@@ -1,30 +1,44 @@
 import React from 'react';
-import { Button, Container, Header, Icon } from 'semantic-ui-react';
+import { withStyles } from 'material-ui/styles';
+import { blueGrey, teal } from 'material-ui/colors';
```

Comment:
As discussed, and one for the refactor, but I think a general rule we should be aiming for is that we never directly reference material ui colours except from a single central place.